packages/terra-navigation-prompt/CHANGELOG.md

```diff
@@ -3,6 +3,9 @@ Changelog
 
 Unreleased
 ----------
+### Breaking Changes
+* Changed 'NavigationPromptCheckpoint' to have the same logic as 'notification-dialog' changes
```

Comment:
Can this change log entry be more specific? What props are changing? Reasoning behind the change?

Reply:
Fixed here https://github.com/cerner/terra-framework/pull/899/commits/cd5d87726ffa459e43be295823a935fea67d1f7a#diff-22332093e32474b48af82c8669855bd2R8